docs/articles/documentation/extending-testcafe/custom-reporter/reporter-project-commands.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Reporter Project Commands
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/reporter-project-commands/
+---
+# Reporter Project Commands
```

Comment:
No-no, even *managing* was more meaningful
I would say that all these four sections relate to *Creating...*